src/web-ui/src/partials/TermsAndConditions/TermsAndConditions.vue

```diff
@@ -0,0 +1,24 @@
+<template>
+  <div class="terms-and-conditions">
+    <p class="mb-1">
+      The Retail Demo Store is a demo application. All content displayed in the Retail Demo Store is fictitious and for
+      demonstration purposes only.
+    </p>
+
+    <p class="mb-1">
+      All images shown are from the open source website, Unsplash.
```

Comment:
Just qq: Do we need to link Unsplash here?

Reply:
As per the offline discussion, please add in the link to Unsplash

Reply:
Done